taxi/views.py

```diff
@@ -87,3 +99,33 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+# ------Car-Driver------------------------------------------------------------
+
+
+@login_required
+def assign_user_to_car(request, pk):
+    car = get_object_or_404(Car, pk=pk)
+
+    if request.user not in car.drivers.all():
+        car.drivers.add(request.user)
+    else:
+        car.drivers.remove(request.user)
+
+    return redirect("taxi:car-detail", pk=pk)
```

Comment:
Ensure that the `assign_user_to_car` function is correctly imported and referenced in the URL configuration. Additionally, consider adding permission checks to ensure that only authorized users can assign or remove drivers from cars.